docs/12-batch-shipyard-configuration-global.md

```diff
@@ -246,9 +245,13 @@ This property is required.
       for the registry in the `singularity_registry` property in the
       credentials file. If this property is empty or is not specified, no
       Singularity images will be pre-loaded on to compute nodes which will
-      lead to increased task startup latency. It is highly recommended not
-      to leave this property empty if possible. Note that `singularity_images`
-      is incompatible with `native` container support enabled pools.
+      lead to increased latency to begin task execution. It is highly
+      recommended not to leave this property empty if possible. Due to
+      Singularity limitations, if the image specified at a certain URI
+      changes, the image will need to be pulled again from the registry the
+      next time that the image is used in a task which will lead to increased
+      latency to begin task execution. Note that `singularity_images` is
```

Comment:
begin task execution, and lead to potential inconsistencies between task executions.